tests/github/w3c/csswg-drafts/cssom-1/Overview.html

```diff
@@ -1898,7 +1898,7 @@ <h4 class="heading settled" data-level="6.6.1" id="the-cssstyledeclaration-inter
    <p>For each CSS property <var>property</var> that is a <a data-link-type="dfn" href="#supported-css-property" id="ref-for-supported-css-property②">supported CSS property</a>,
 the following partial interface applies where <var>camel-cased attribute</var> is obtained by running the <a data-link-type="dfn" href="#css-property-to-idl-attribute" id="ref-for-css-property-to-idl-attribute">CSS property to IDL attribute</a> algorithm for <var>property</var>.</p>
 <pre class="idl extract highlight def"><c- b>partial</c-> <c- b>interface</c-> <a class="idl-code" data-link-type="interface" href="#cssstyledeclaration" id="ref-for-cssstyledeclaration④"><c- g>CSSStyleDeclaration</c-></a> {
-  [<a class="idl-code" data-link-type="extended-attribute" href="https://html.spec.whatwg.org/multipage/custom-elements.html#cereactions" id="ref-for-cereactions④"><c- g>CEReactions</c-></a>] <c- b>attribute</c-> [<a class="idl-code" data-link-type="extended-attribute" href="https://heycam.github.io/webidl/#TreatNullAs" id="ref-for-TreatNullAs④"><c- g>TreatNullAs</c-></a>=<a class="n" data-link-type="idl-name"><c- n>EmptyString</c-></a>] <a class="n" data-link-type="idl-name" href="#cssomstring" id="ref-for-cssomstring②⑤"><c- n>CSSOMString</c-></a> <dfn class="idl-code" data-dfn-for="CSSStyleDeclaration" data-dfn-type="attribute" data-export data-type="[TreatNullAs=EmptyString] CSSOMString" id="dom-cssstyledeclaration-_camel_cased_attribute"><code><c- g>_camel_cased_attribute</c-></code><a class="self-link" href="#dom-cssstyledeclaration-_camel_cased_attribute"></a></dfn>;
+  [<a class="idl-code" data-link-type="extended-attribute" href="https://html.spec.whatwg.org/multipage/custom-elements.html#cereactions" id="ref-for-cereactions④"><c- g>CEReactions</c-></a>] <c- b>attribute</c-> [<a class="idl-code" data-link-type="extended-attribute" href="https://heycam.github.io/webidl/#TreatNullAs" id="ref-for-TreatNullAs④"><c- g>TreatNullAs</c-></a>=<a class="n" data-link-type="idl-name"><c- n>EmptyString</c-></a>] <a class="n" data-link-type="idl-name" href="#cssomstring" id="ref-for-cssomstring②⑤"><c- n>CSSOMString</c-></a> <dfn class="idl-code" data-dfn-for="CSSStyleDeclaration" data-dfn-type="attribute" data-export data-lt="camel_cased_attribute" data-type="[TreatNullAs=EmptyString] CSSOMString" id="dom-cssstyledeclaration-camel_cased_attribute"><code><c- g>_camel_cased_attribute</c-></code><a class="self-link" href="#dom-cssstyledeclaration-camel_cased_attribute"></a></dfn>;
```

Comment:
This adds `data-lt="camel_cased_attribute"` and removes some underscores, but that seems OK. The actual visible text still has the underscore.